### DISCUSSION TITLE
[Feature] Add a description field for people

### DISCUSSION THREAD

**Discussion by imler** [immich-app/immich#8679] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Each person should have a dedicated field with a description - similar to album descriptions.

The purposes of this field are endless and everyone could put there whatever they want about that specific person - e.g. family connections, date of death, place of leaving, fun facts, etc.

I have a big collection of family photos inherited from my ancestors and I have many people tagged, but after some time it’s hard to remember all the connections between them (especially regarding people I didn’t have a chance to meet). That description field directly inside Immich would be helpful to quickly recall information about a specific person.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4872
